src/System.Private.CoreLib/shared/System/Text/ValueStringBuilder.cs

```diff
@@ -139,6 +139,21 @@ public void Insert(int index, char value, int count)
             _pos += count;
         }
 
+        public void Insert(int index, string s)
+        {
+            int count = s.Length;
+
+            if (_pos > (_chars.Length - count))
+            {
+                Grow(count);
+            }
+
+            int remaining = _pos - index;
+            _chars.Slice(index, remaining).CopyTo(_chars.Slice(index + count));
+            s.AsSpan().CopyTo(_chars.Slice(index, count));
```

Comment:
Nit: looks like this could just be `s.AsSpan().CopyTo(_chars.Slice(index));`

Reply:
Fixed.